examples/datadog/agent_on_openshift_values.yaml

```diff
@@ -6,25 +6,15 @@
 # constraints, see https://docs.datadoghq.com/integrations/openshift/ and
 # https://www.datadoghq.com/blog/openshift-monitoring-with-datadog/
 
-targetSystem: "linux"
 datadog:
-  # apiKey: <DATADOG_API_KEY>
-  # appKey: <DATADOG_APP_KEY>
-  # If not using secrets, then use apiKey and appKey instead
-  apiKeyExistingSecret: <DATADOG_API_KEY_SECRET>
-  appKeyExistingSecret: <DATADOG_APP_KEY_SECRET>
+  apiKey: <DATADOG_API_KEY>
+  appKey: <DATADOG_APP_KEY>
   clusterName: <CLUSTER_NAME>
   tags: []
-  criSocketPath: /var/run/crio/crio.sock
   # Depending on your DNS/SSL setup, it might not be possible to verify the Kubelet cert properly
   # If you have proper CA, you can switch it to true
   kubelet:
     tlsVerify: false
-  confd:
```

Comment:
just confirming that we are removing it because the check is now autodetected?

Reply:
Yes we wanted to keep it simple by removing unnecessary fields. As I tested in my sandbox, it seems working properly without it. 
But do you think it better to keep it?